lib/user.ts

```diff
@@ -1,14 +1,8 @@
 /// <reference types="@nextcloud/typings" />
 
-declare var OC: Nextcloud.v16.OC
-	| Nextcloud.v17.OC
-	| Nextcloud.v18.OC
-	| Nextcloud.v19.OC
-	| Nextcloud.v20.OC
-	| Nextcloud.v21.OC
-	| Nextcloud.v22.OC
-	| Nextcloud.v20.OC
-	| Nextcloud.v24.OC;
+declare var OC: Nextcloud.v23.OC
+	| Nextcloud.v24.OC
+	| Nextcloud.v25.OC;
```

Comment:
26 ?

Reply:
v26 is not provided by `@nextcloud/typings`

Reply:
*yet*

Reply:
That was my point yes :wink: